public_suffix_list.dat

```diff
@@ -15755,4 +15755,8 @@ basicserver.io
 virtualserver.io
 enterprisecloud.nu
 
+// is-cod.in : https://is-cod.in
+// Submitted by Araan Sheikh <dotiscodin@gmail.com>
+is-cod.in
+
```

Comment:
Fix your sorting please. I would recommend reading the guidelines as it is evident you have not.

Reply:
okay, no time to read each and every sentences lol

Reply:
If you aren't able to follow the guidelines and rectify the issues I've listed, you won't be able to be approved. If you don't have time to follow and read the guidelines, I would recommend closing this PR as it won't be approved.

Reply:
Ig it's better to close the pull request right now, will be back when we have 1000+ users.